ssl/internal.h

```diff
@@ -2429,6 +2444,15 @@ bool tls12_add_verify_sigalgs(const SSL_HANDSHAKE *hs, CBB *out);
 bool tls12_check_peer_sigalg(const SSL_HANDSHAKE *hs, uint8_t *out_alert,
                              uint16_t sigalg);
 
+// tls13_get_verify_sigalgs_pha calls |tls12_get_verify_sigalgs| for the given
+// handshake object. It takes the sigalgs returned, creates a deep copy, and
+// returns the copy. These are used for PHA .
+OPENSSL_EXPORT Span<const uint16_t> tls13_get_verify_sigalgs_pha(const SSL_HANDSHAKE *hs);
```

Comment:
It might be worth calling out the issues you ran into and why we need to make a deep copy for PHA here :)